docs/mastering-plone-5/instructions_plone5/index.md

```diff
@@ -0,0 +1,382 @@
+```
+
+If the output says `Serving on http://0.0.0.0:8080` then you are in business.
+
+If you point your browser at <http://localhost:8080> you see that Plone is running.
+
+```{figure} _static/instructions_plone_running.png
+:alt: A running Plone instance.
+:scale: 50 %
+
+A running plone instance.
+```
+
+There is no Plone site yet - we will create one in chapter 6.
+
+Now you have a working Plone site up and running and can continue with the next chapter.
+
+You can stop the running instance anytime using {kbd}`ctrl + c`.
+
+```{warning}
+If there is an error message you should either try to fix it or use vagrant and continue in this chapter.
+```
+
+(plone5-instructions-vagrant-label)=
+
+## Installing Plone with Vagrant
+
+We use a virtual machine (Ubuntu 18.04) to run Plone during the training.
+
+We rely on [Vagrant](https://www.vagrantup.com) and [VirtualBox](https://www.virtualbox.org) to give the same development environment to everyone.
+
+[Vagrant](https://www.vagrantup.com) is a tool for building complete development environments.
+
+We use it together with Oracle’s [VirtualBox](https://www.virtualbox.org) to create and manage a virtual environment.
+
+(plone5-install-virtualbox)=
+
+### Install VirtualBox
+
+Vagrant uses Oracle’s VirtualBox to create virtual environments.
+
+Here is a link directly to the download page: <https://www.virtualbox.org/wiki/Downloads>.
+
+We use VirtualBox 6.0.x
+
+(plone5-instructions-configure-vagrant-label)=
+
+### Install and configure Vagrant
+
+Get the latest version from <https://developer.hashicorp.com/vagrant/install> for your operating system and install it.
+
+Now your system has a command {command}`vagrant` that you can run in the terminal.
+
+First, create a directory in which you want to do the training.
+
+```{warning}
+If you already have a {file}`training` directory because you followed the **Installing Plone without vagrant** instructions above,
+you should either delete it, rename it, or use a different name below.
+```
+
+```shell
+mkdir training
+cd training
+```
+
+Setup Vagrant to automatically install the current guest additions.
+You can choose to skip this step if you encounter any problems with it.
+
+```shell
+vagrant plugin install vagrant-vbguest
+```
+
+Now download {download}`plone_training_config.zip <../../_static/plone_training_config.zip>` and copy its contents into your training directory.
+
+```shell
+wget https://github.com/plone/training/raw/refs/heads/main/docs/_static/plone_training_config.zip
+unzip plone_training_config.zip
+```
+
+The training directory should now hold the file {file}`Vagrantfile` and the directory {file}`manifests` which again contains several files.
+
+Now start setting up the virtual machine (VM) that is configured in {file}`Vagrantfile`:
+
+```shell
+vagrant up
+```
+
+This takes a **veeeeery loooong time** (between 10 minutes and 1h depending on your Internet connection and system speed) since it does all the following steps:
```

Comment:
```suggestion
This takes a **veeeeery loooong time** (between 10 minutes and 1 hour depending on your Internet connection and system speed) since it does all the following steps:
```